src/main/policy_config.c

```diff
@@ -531,6 +565,134 @@ as_status set_admin_policy(as_policy_admin *admin_policy, PyObject *py_policy)
     return AEROSPIKE_OK;
 }
 
+// For batch write, batch apply, and batch remove policies:
+// Don't set expressions field since it depends on the client's
+// serialization policy
```

Comment:
I think you should note that the client level policies do not accept expressions in the docs at https://aerospike-python-client.readthedocs.io/en/latest/aerospike.html#client-config

Reply:
Done

Reply:
I added a docs preview at the top